proxy/envoy/config.go

```diff
@@ -102,59 +102,35 @@ func Generate(instances []*model.ServiceInstance, services []*model.Service,
 		},
 		ClusterManager: ClusterManager{
 			Clusters: clusters,
-			SDS: SDS{
-				Cluster:        buildSDSCluster(mesh),
+			SDS: &SDS{
+				Cluster:        buildDiscoveryCluster(mesh.DiscoveryAddress, "sds"),
+				RefreshDelayMs: 1000,
```

Comment:
Please add a TODO note here to make these three fields configurable (sds, cds, rds)